src/app/observation-domain/observation-domain.component.html

```
@@ -2,29 +2,29 @@
         <button mat-icon-button (click)="downloadPop()" class="download-btn">
             <mat-icon>arrow_downward</mat-icon>
           </button>
           <p class="download-text">{{'DOWNLOAD'|translate}}</p>
     </div>
 </div>
 
-<div *ngIf="loaded" class="domain-content">
-  @if(evidences?.length > 0){
-    <div class="domain-container" *ngFor="let entity of evidences; let i = index">
+<div *ngIf="loaded(); else loadingTpl" class="domain-content">
+  @if(evidences().length > 0){
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)